shimming-toolbox/shimmingtoolbox/shim/sequencer.py

```diff
@@ -1946,40 +2116,43 @@ def plot_full_time_std(self, unshimmed, masked_shim_static_riro, mask_fmap_cs, m
         fig.savefig(fname_figure, bbox_inches='tight')
 
 
-def plot_full_mask(unshimmed, shimmed_masked, mask, path_output):
+def plot_full_mask(unshimmed, shimmed_masked, mask, softmask, path_output):
```

Comment:
I would only input the "softmask" and process the mask from it in the function. My rule of thumb is to not include an input if I can compute it from another input. The notion should be named mask even though it might be a soft mask because it could also be a binary mask.